sidecar2cdxj.py

```diff
@@ -16,27 +15,23 @@ def create_cdxj_path(sidecar_file, archive_dir):
     return os.path.join(archive_dir, cdxj_file)
 
 
-def convert_payload_to_dict(record):
+def convert_payload_to_json(record):
     """Parse a record's payload, put the fields into a dictionary and return the dictionary."""
     string_payload = record.content_stream().read().decode('utf-8')
     payload_list = string_payload.split('\n')
     new_dict = {}
     for item in payload_list:
         item_key, value = item.split(': ', 1)
-        try:
-            new_value = ast.literal_eval(value)
-            new_dict[item_key] = new_value
-        except ValueError:
-            new_dict[item_key] = value
-    return new_dict
+        new_dict[item_key] = json.loads(value)
```

Comment:
We may need to wrap this in a try/except to catch a `json.decoder.JSONDecodeError` or whatever is relevant in the case the value is a plain string, not a dict.